docs/example-scenario/apps/sap-dev-test.md

```diff
@@ -0,0 +1,109 @@
+---
+title: SAP for a dev/test environment
+description: SAP scenario for a dev/test environment
+author: AndrewDibbins
+ms.date: 7/11/18
+---
+
+# SAP for a dev/test workload
+
+This architectural design shows a set of proven practices for running SAP NetWeaver in a Windows or Linux environment on Azure. The database is AnyDB, the SAP term for any supported DBMS (that isn't SAP HANA). This architecture is deployed with specific virtual machine (VM) sizes that can be changed to accommodate your organization's needs.
+
+## Related use cases
+
+Consider this scenario for the following use cases:
+
+* Non-Critical SAP non-productive workloads (Sandbox, Development, Test, Quality Assurance)
+* Non-critical SAP Business One workloads
+
+## Architecture
+
+![Diagram](media/sap-2tier/SAP-Infra-2Tier_finalversion.png)
```

Comment:
Does a dev/test deployment need to use a hub-spoke network?

Reply:
Should the diagram be numbered?

Reply:
I think we'll have to skip on fixing this right now, I'm getting the orange triangles removed, beyond that we'll have to wait to fix this.